README.md

```diff
@@ -1,42 +1,91 @@
 
-The `-p` option takes a time interval in hours, days, months, etc. E.g. `-p "10 weeks"` will only count participation in ballots that were created within the last 10 weeks. Alternatively, instead of a _time_, you can specify the earliest block _number_ as a decimal integer with the `-b` option.
+## Dependencies
+Prior to downloading poa-ballot-stats, install and activate a fully synchronized node connected to the POA network. See the [POA installation guide](https://github.com/poanetwork/wiki/wiki/POA-Installation) for instructions.
 
-Examples:
+**Note:** poa-ballot-stats must access the full network logs. Use these flags when running a node `--pruning=archive --no-warp`
+
+**Example:**
+```bash
+$ parity --chain c:\path\to\spec.json --reserved-peers c:\path\to\bootnodes.txt --pruning=archive --no-warp
+```
+
+## Installation
+
+### Stable Release
+
+Download the archive for your platform from the latest [release](https://github.com/poanetwork/poa-ballot-stats/releases) and unpack. Run the tool with `./poa-ballot-stats <options>`.
+
+#### Options
+
+`-h, --help` view command line options and help information. 
+
+`<url>` specify a different endpoint if your node uses a non-standard port. The default connects to a local node `http://127.0.0.1:8545`.
+
+`-V, --version` prints version information.
+
+
+`-v, --verbose` display collected ballot and key change events and the list of participating and abstaining voters for each ballot.
+
+`-c, --contracts <contracts>`  append a map with POA contract addresses in JSON format. The current maps for the main and test network are in the `contracts` folder. Default is the main network `core.json` file.
+
+`-p, --period <period>`  a time interval in hours, days, months, etc. For example, `-p "10 weeks"` only counts participation in ballots created within the last 10 weeks. 
+
+`-b, --block` takes the earliest block _number_ as a decimal option. For example, `-b 524647` counts participation from block number 524647 onward.
+
+
+**Examples:**
 
 ```bash
-$ ./poa-ballot-stats -h
 $ ./poa-ballot-stats
+$ ./poa-ballot-stats -h
 $ ./poa-ballot-stats https://core.poa.network -v -p "10 weeks"
 $ ./poa-ballot-stats -c contracts/sokol.json https://sokol.poa.network -v
 ```
 
-## Latest code
+### Latest code
 
 If you have a recent version of [Rust](https://www.rust-lang.org/), you can clone this repository and use `cargo run --` instead of `./poa-ballot-stats` to compile and run the latest version of the code.
 
-## Screenshot
+## Troubleshooting
 
-![Screenshot](screenshot.png)
+### No Events Found error
+
+1.	Parity must be fully synced to the correct node and running in full mode, not "light" mode. Check Parity UI and/or Task Manager to confirm Parity is synced and actively connected to peers.
+
+2.	The correct network must be selected in Parity. The Parity UI will show the current network selection in green. Make sure this is the correct network, and not the Foundation or other Etherium network.
```

Comment:
Here we could also mention again that you need to use a contracts address file (`-c` option) that matches the network. Included are files for the main POA network ("core") and the POA test network ("sokol"). `poa-ballot-stats` needs to be run with a file matching the network that the Parity node is connected to.